homeassistant/components/velbus/climate.py

```diff
@@ -18,6 +18,11 @@
 from .const import DOMAIN, PRESET_MODES
 from .entity import VelbusEntity, api_call
 
+THERMOSTAT_MODE_MAP: dict[str, HVACMode] = {
+    """Set up Velbus switch based on config_entry.""" "heat": HVACMode.HEAT,
+    "cool": HVACMode.COOL,
+}
+THERMOSTAT_INV_MODE_MAP = {v: k for k, v in THERMOSTAT_MODE_MAP.items()}
```

Comment:
You're not using them at the moment. Also because HVACMode.COOL already means "cool" so there is no real need to to a conversion for the Velbus lib to understand what you mean.

Be aware, the moment that Velbus introduces HVAC modes that are not 1:1 mapped, we need this again